doc/introduction.rst

```diff
@@ -89,8 +85,72 @@ Why is machine learning relevant to NeuroImaging? A few examples!
 Installing nilearn
 ====================
 
-.. raw:: html
-   :file: install_doc_component.html
+.. tab-set::
+
+  .. tab-item:: Pip install
+
+      **1. Download and install 64-bit Anaconda**
```

Comment:
also python and nilearn work fine on 32 bit architectures

Reply:
It's good that users use a custom python env though, don't you think? Do you want to recommend miniconda instead?

Reply:
> the main page and quickstart look great. I'm not sure about the choice of featured examples, I think some could be removed, but that's probably a discussion for another issue

Yes, let's keep it for another PR maybe :)

> * some dependencies seem to be missing from the list: lxml, requests. (that is not really related to this pr, we should list dependencies in setup.cfg only and link to it in the readme)

It has be resolved since you posted this I believe

> * link to how to contribute page is an absolute URL, most other links I saw in main page & user guide use relative urls

This is because `quickstart` is merely a copy of the `readme.md`, which is our github frontpage, so urls need to be absolute.

> also in the [examples page section 3](https://nilearn-doc-dev.github.io/auto_examples/index.html#functional-connectivity) a sphinx command is shown verbatim ":term:`Dictionary learning`" and in https://nilearn-doc-dev.github.io/plotting/index.html images appear squished (too narrow)

I removed the term tag in the title. Besides, I think I fixed the squished images problem.